orchestrator/careplanservice/handle_get.go

```diff
@@ -15,17 +15,27 @@ import (
 func (s *Service) handleGet(httpResponse http.ResponseWriter, request *http.Request) error {
 	queryParams := request.URL.Query()
 	urlParts := strings.Split(strings.TrimPrefix(request.URL.Path, basePath+"/"), "/")
+	headers := new(fhirclient.Headers)
+
 	switch urlParts[0] {
 	case "CarePlan":
 		// fetch CarePlan + CareTeam, validate requester is participant of CareTeam
 		// if there are query params present, we need to handle the request a bit differently
 		if len(queryParams) != 0 {
-			if queryParams.Get("_id") == "" {
-				return coolfhir.NewErrorWithCode("Query parameter _id is not present, can't find CarePlan", http.StatusBadRequest)
+			// TODO: Check if all parameters are allowed
+			// TODO: Is it acceptable to return the CareTeam in the bundle if it is not requested?
```

Comment:
If it's a Search operation, a Bundle is expected, then it might be semantically acceptable. But I'd think it would be weird and don't understand why we would need to.

Reply:
Discussed with Bram, should be fine